templates/rhpam75-kieserver-externaldb.yaml

```diff
@@ -203,7 +203,7 @@ parameters:
   required: false
 ## Extension image configuration BEGIN
 - displayName: Drivers Extension Image
-  description: ImageStreamTag definition for the image containing the drivers and configuration. i.e custom-driver-image:1.0.
+  description: ImageStreamTag definition for the image containing the drivers and configuration. For example, custom-driver-image:1.0.
```

Comment:
Most everywhere else you say "... , for example, ..." Best to do the same here I suppose.

Reply:
It would be a bit less clear (as ", for example" could also be read as next to "configuration") and there is this same format in one more place (MAVEN_REPO_ID), so leaving this one be. I did change the other one, though.